### PR TITLE
[Feature] Update Stylelint to version 14.0

### PR DESCRIPTION
**Pull request by antoine4livre**
### Modified
- Configuration to use `"postcss-scss"` as `customSyntax`

### Updated
- `stylelint` package
- `stylelint-order` & `stylelint-scss` packages
- `@studiometa/eslint-config` & `eslint` packages

### Added
- `postcss` & `postcss-scss` packages

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,10 +1,6 @@
 module.exports = {
-  extends: [
-    'stylelint-config-recommended',
```
Comment (titouanmathis) [inline index.js:3]: **question:** Why are the `stylelint-config-recommended` and `stylelint-config-recommended`scss` packages removed? 🤔

Comment (antoine4livre) [reply index.js:3]: I thought this was already included in the new configuration by following the documentation.

Comment (titouanmathis) [reply index.js:3]: I did not see any mention of this packages in the [migration guide](https://github.com/stylelint/stylelint/blob/14.0.0/docs/migration-guide/to-14.md) I found, or maybe I misread something?

Comment (antoine4livre) [reply index.js:3]: Precisely! I will put them in the configuration again.

**Pull request merged**